csuntechlab/iSTART
# STR-145 -- FE: Implement Information Slide #20 with carousel

# Description
Implemented bootstrap carousel and implemented it in slide 20.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. Visit alcohol module.
3. Click continue until you reach the `carousel informational slide`.
4. Ensure all functionality works.
5. Ensure mobile responsiveness and other visual aesthetics.
